db/migrate/20200508194824_create_cached_uphold_tips.rb

```diff
@@ -0,0 +1,14 @@
+class CreateCachedUpholdTips < ActiveRecord::Migration[6.0]
+  def change
+    create_table :cached_uphold_tips, id: :uuid, default: -> {"uuid_generate_v4()"} do |t|
+      t.references :uphold_connection_for_channel
+      t.uuid :transaction_id, index: true
+      t.string :amount
+      t.string :settlement_currency
+      t.string :settlement_amount
+      t.datetime :uphold_created_at, index: true
```

Comment:
I think you should remove this index and use a multi column index with UPFC ID as the first index parameter

Reply:
https://github.com/brave-intl/publishers/pull/2682/commits/053d757f795eae429d0bba8a67beab05ed218c8e